### ISSUE TITLE
[P1][Sourcing][Accessibility][Vertical Navigation]List label read twice when reaching the list first time using tab key

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10387]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
List label read twice when reaching the list first time using tab key

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
replicable on https://fundamental-ngx.netlify.app/#/core/vertical-navigation
As a part of evaluation of the issue https://github.com/SAP/fundamental-ngx/issues/9608 , this new issue is found

#### If this is a bug, please provide steps for reproducing it.
1) Reach the list using tab key for the first time
2) notice the screenreader reading the label "Overview" twice

#### Please provide relevant source code if applicable.

https://github.com/SAP/fundamental-ngx/assets/88182358/76a7c8af-ccc5-41cb-8946-39b0530b6f3e

**Comment by droshev** [comment]
We need to downport it `ng15`.

**Comment by N1XUS** [comment]
Hello @I543348. From what I understand, its a default behaviour of the VoiceOver functionality. Same behaviour can be observed in UI5 example: https://sapui5.hana.ondemand.com/#/entity/sap.tnt.ToolPage/sample/sap.tnt.sample.ToolPage

**Comment by droshev** [comment]
fixed with `0.43.8`

**Comment by sagar7259** [comment]
Hello,

Now the title 'Overview' is being read only once on using tab key to reach this item. Hence this ticket can be closed as its fixed now.

Regards,
Sagar
